src/code/modules/notifications.js

```diff
@@ -10,6 +10,10 @@ export const GUIDE_ERRORED = "Guide errored";
 export const ADVANCE_NOTIFICATIONS = "Advance notifications";
 export const CLOSE_NOTIFICATIONS = "Close notifications";
 
+export const notificationType = {
+  DIALOG: "dialog"
```

Comment:
1. Yes, narrative might be better. I'll change that.
2. I think the other types (of which currently there seem to be two kinds, ITS-messages and our own canned feedback messages) could be named once we need to distinguish them. So if we don't want the ITS to append to an existing feedback message, for example, but we do want it to append to an existing ITS message, then we could label them. (They could also have been labeled now, granted.)